docs/docs/reference/experimental/canthrow.md

```diff
@@ -47,9 +47,9 @@ However, a programming language is not a framework; it has to cater also for tho
 Why does `map` work so poorly with Java's checked exception model? It's because
 `map`'s signature limits function arguments to not throw checked exceptions. We could try to come up with a more polymorphic formulation of `map`. For instance, it could look like this:
 ```scala
-  def map[B, E](f: A => B canThrow E): List[B] canThrow E
+  def map[B, E](f: A => B throws E): List[B] throws E
 ```
-This assumes a type `A canThrow E` to indicate computations of type `A` that can throw an exception of type `E`. But in practice the overhead of the additional type parameters makes this approach unappealing as well. Note in particular that we'd have to parameterize _every method_ that takes a function argument that way, so the added overhead of declaring all these exception types looks just like a sort of ceremony we would like to avoid.
+This assumes a type `A throws E` to indicate computations of type `A` that can throw an exception of type `E`. But in practice the overhead of the additional type parameters makes this approach unappealing as well. Note in particular that we'd have to parameterize _every method_ that takes a function argument that way, so the added overhead of declaring all these exception types looks just like a sort of ceremony we would like to avoid.
```

Comment:
FYI, we are moving the Scala 3 documentation out of dotty.epfl.ch. So, we should preferably update the reference [here](https://github.com/scala/docs.scala-lang/tree/main/_scala3-reference) instead (you can see the result [here](https://docs.scala-lang.org/scala3/reference/overview.html)). I know this is inconvenient because we have te create two PRs when we do one change: one PR in the compiler, and one PR in the reference documentation. This is the best we have so far, but VirtusLab is working on a way to move back the sources of the documentation in this repo (and render them in docs.scala-lang.org, still).

BTW, I see that this page is currently orphaned: there is no link to it in the sidebar: https://dotty.epfl.ch/docs/reference/experimental/canthrow.html

Reply:
Yes, it would be much better if we could keep one PR!

Pages about experimental features are currently not linked. We need something like [the Rust unstable book](https://doc.rust-lang.org/beta/unstable-book/the-unstable-book.html)

Reply:
>  We need something like [the Rust unstable book](https://doc.rust-lang.org/beta/unstable-book/the-unstable-book.html)

Well, this would be very easy to achieve, just by linking them in a chapter “experimental features”, no?

Reply:
A different solution would be a simple github action that updates the other repo whenever the docs in this one change